packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -385,13 +385,14 @@ library ChainAssertions {
 
         if (_isProxy) {
             require(address(portal.disputeGameFactory()) == _contracts.DisputeGameFactory, "CHECK-OP2-20");
+            require(address(portal.anchorStateRegistry()) == _contracts.AnchorStateRegistry, "CHECK-OP2-25");
```

Comment:
We will want similar updates in `StandardValidator`. I think we can now drop the V200 suffix and just update StandardValidator in place like we do with OPCM, but let's confirm with @mslipper

Reply:
Ok, will update in StandardValidator